browser-interface/packages/shared/loading/ReportFatalError.ts

```diff
@@ -87,8 +87,6 @@ export function BringDownClientAndReportFatalError(
 ) {
   let sagaStack: string | undefined = payload['sagaStack']
 
-  debugger
```

Comment:
Why are we removing this debugger? It's useful for debugging. The `debugger` only triggers if you have the console opened.

Reply:
> Testing it with 2 tabs... what happened is both tabs got to the same state. They both show the error, they keep playing the music (from Genesis Plaza) and they both keep high activity in the console while trying to reconnect. ![Screenshot](https://user-images.githubusercontent.com/959563/228610842-18a5f4f8-b97b-4492-a773-7e0070ac6ff1.png)

Yes, I see two things with the current approach:
1. It takes some time before clients realize they need to go down
2. There's still reconnecting logic connected but that due to having a fatal error set, [should not be actually invoking the connection logic here](https://github.com/decentraland/unity-renderer/blob/hotfix/brind-down-client-on-id-duplicated/browser-interface/packages/shared/comms/sagas.ts#L413)